deploy/kubernetes/base/controller_setup/controller.yaml

```diff
@@ -82,5 +84,10 @@ spec:
         - name: cloud-sa-volume
           secret:
             secretName: cloud-sa
+      # https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
```

Comment:
From the other PR, I think we should not include this otherwise the controller won't restart if the node becomes unhealthy. It's only really meant for DaemonSets.

Reply:
right, forgot to update it.